.vim/coc-settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think we need the editor config.

Reply:
oops i didnt notice that sorry